.github/workflows/atomicdex-desktop-ci.yml

```diff
@@ -181,11 +193,11 @@ jobs:
           echo "CC=clang-12" >> $GITHUB_ENV
 
       - name: vcpkg deps (All)
-        uses: lukka/run-vcpkg@v10
+        uses: lukka/run-vcpkg@v11
         with:
           vcpkgDirectory: '${{ github.workspace }}/ci_tools_atomic_dex/vcpkg-repo'
-          appendedCacheKey: ${{ hashFiles('vcpkg.json') }}
           vcpkgJsonGlob: 'vcpkg.json'
+          appendedCacheKey: vcpkginstall
```

Comment:
saw this just now:
```
Warning: Unexpected input(s) 'appendedCacheKey', valid inputs are ['vcpkgDirectory', 'runVcpkgInstall', 'vcpkgGitCommitId', 'vcpkgGitURL', 'doNotUpdateVcpkg', 'doNotCache', 'vcpkgJsonGlob', 'vcpkgJsonIgnores', 'binaryCachePath', 'runVcpkgFormatString', 'useShell', 'logCollectionRegExps']
```
i have no idea what it does but seems like `runVcpkgInstall` would be a "correct" value

Reply:
the way caching works changed, so the old key is obsolete. `runVcpkgInstall` should remain false (as the default)